GREEDY/Brands.txt

```diff
@@ -0,0 +1,10 @@
+﻿iphone
```

Comment:
Do we have to save all this stuff in .txt files (seems unprofessional) or is it just for now?